lib/render/pxrUsdMayaGL/batchRenderer.cpp

```diff
@@ -130,6 +140,13 @@ UsdMayaGLBatchRenderer::GetDelegatePrefix(const bool isViewport2) const
 bool
 UsdMayaGLBatchRenderer::AddShapeAdapter(PxrMayaHdShapeAdapter* shapeAdapter)
 {
+    TRACE_FUNCTION();
+
+    MProfilingScope profilingScope(
+        ProfilerCategory,
+        MProfiler::kColorG_L3,
```

Comment:
Is there a common convention in the colors/levels used with the Profiler? I picked `G` fairly arbitrarily just because it was the last one. Then for levels, I used `1` for the "main" Maya API callback entry points in the `MPxSurfaceShapeUI`/`MHWRender::MPxDrawOverride` overrides, `2` for the proxy shape adapter as a kind of middle tier, and then `3` for the batch renderer itself at the lowest level.

Let me know if there's a better pattern for that!

Reply:
This should help - https://knowledge.autodesk.com/support/maya/learn-explore/caas/CloudHelp/cloudhelp/2019/ENU/Maya-Customizing/files/GUID-54095CD4-7C0E-4031-8981-A45131348182-htm.html

Shades can be used to make the reading of stacked events easier, but we didn't enforce any rule for it.

Reply:
In general - blue should be reserved for rendering tasks, green for compute, and purple for dirty propagation.

Reply:
Ahh, thanks @kxl-adsk! This linked page had some helpful examples too:
https://help.autodesk.com/view/MAYAUL/2019/ENU/?guid=GUID-0FF1BA78-0493-463F-8E66-8014FB0F47BC

It took a bit of trial and error though to figure out the mapping of `ProfilingColor` enum value to display color though:
```
kColorA = Purple
kColorB = Green
kColorC = Blue
kColorD = Orange
kColorE = Yellow
kColorG = Gray
```

I'm about to push some new commits with a more "compliant" color scheme with yellows for render-related compute and drawing prep and blues for rendering.